contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_not_be_null_and_column_to_not_be_empty.py

```diff
@@ -322,8 +322,8 @@ def _validate(
         runtime_configuration: Optional[dict] = None,
         execution_engine: Optional[ExecutionEngine] = None,
     ):
-        result_format = self.get_result_format(
-            configuration=configuration, runtime_configuration=runtime_configuration
```

Comment:
we're safe to remove the `configuration` parameter from `_validate` now, right?

Reply:
I think so but not confident - gonna do that in a follow-up effort since it is more copmlex